libs/extensions/src/NanoGenParticle.cpp

```diff
@@ -6,7 +6,14 @@
 
 using namespace std;
 
-int NanoGenParticle::GetPdgId() { return physicsObject->Get("pdgId"); }
+float NanoGenParticle::GetDxy(float pv_x, float pv_y) {
```

Comment:
Did you remove GetPdgId() method?

Reply:
No but I moved it to NanoGenParticle.hpp as it's just one line like we've done in the other nano-classes